packages/api-headless-cms/src/content/plugins/schema/createManageResolvers.ts

```diff
@@ -27,11 +27,15 @@ export const createManageResolvers: CreateManageResolvers = ({
 }) => {
     const typeName = createTypeName(model.modelId);
     const mTypeName = createManageTypeName(typeName);
+    let pluralizedTypeName = pluralize(typeName);
```

Comment:
1. We could write this like so and make it a bit more efficient:

```ts
let pluralizedTypeName;
if (typeName.length === 1) {
  pluralizedTypeName = `${typeName}s`;
} else {
  pluralizedTypeName = pluralize(typeName);
}
```

Bottom line is, we can call the pluralize function only if actually needed.

2. Since we are c/p-ing this code into several places, let's extract it into a separate helper function.

You can put it into the `utils` folder (check imports at the top of the file, e.g. `packages/api-headless-cms/src/content/plugins/schema/createManageResolvers.ts:7`).

3. Add a short comment in it, why we are doing this.

4. Write a short test please.